### ISSUE TITLE
Bonus/Deal page edits - (Incorrect green on "Use Deal" button)

### ISSUE THREAD

**Issue by tomtheall** [ventuso/Fi-Test#101]
1) There is a lime green on the Use Deal/Redeem button. Should be same green as all the other green buttons. I think it's #85bc40

2) Also, for consistency, let's make the "redeem" button to be a rounded corner button, like the other buttons (connect, etc.)

3) Lets display all pictures full width, and have a set ratio for all deal banners. Wide format, so that no matter what image the user uploads it will crop to a 6 : 3.5 ratio (750x438 on iPhone 6)

4) The text could probably be a little smaller overall.

![img_4744](https://cloud.githubusercontent.com/assets/11476339/7258854/c37ec2e4-e824-11e4-8c49-cb7d8509f163.PNG)

**Comment by baonguyenventuso** [comment]
@tomtheall 
Hi Tom,

We got problem when doing center crop for the image, here you go:
1. For example, user make an advertisement banner with text on the image. If you do an automatic center crop, it can cut off some text and user won't able to see it until it go to the mobile. Beside, our mobile would go to detect any kind of devices and its screen size, think about if its Android. 
2. Solution for this problem is allow user to upload an exact image from Portal by giving them a Crop tool (See the way they do with Avatar on Facebook). For this, please give us a list of image ratio for each place we have image (avatar, deals/bonus, background screen) 
3. Same for issue #91 . The avatar of each Merchant should also get crop as user want from Portal, not trying to scale on Phone. 

Best regards,

**Comment by tomtheall** [comment]
Yes I agree. That makes sense, user should crop in the portal and get a preview of how it looks cropped.
I will get those ratios (from the psd file).

- I know all network/user profile avatars are 1:1 (square)

- The network backgrounds I have been preparing are 1080x1920 (9:16 ratio).

- The deals are 6 : 3.5 ratio. We can use the same picture for deal list page and deals detail page.

**Comment by nhanventuso** [comment]
I fixed 1,2,4 on iOS version 1.0.46.

**Comment by dangtran89** [comment]
android fixed in 1.0.30
